From 3755aa0f4cd32e25742069a1a92f08dba6f1dec9 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@gnome.org>
Date: Thu, 21 Jul 2016 18:23:29 -0500
Subject: tests: Rename function to avoid conflict with stdio.h

I guess stdio.h is being included somewhere it previously wasn't by
some header file somewhere.

https://bugs.freedesktop.org/show_bug.cgi?id=97034

diff --git a/tests/dbus/test-tpl-log-walker.c b/tests/dbus/test-tpl-log-walker.c
index d8b5b9f..3b1baf9 100644
--- a/tests/dbus/test-tpl-log-walker.c
+++ b/tests/dbus/test-tpl-log-walker.c
@@ -169,7 +169,7 @@ rewind_cb (GObject *source,
 
 
 static void
-rewind (WalkerTestCaseFixture *fixture,
+rewind_sync (WalkerTestCaseFixture *fixture,
     TplLogWalker *walker,
     guint num_events)
 {
@@ -350,37 +350,37 @@ test_rewind (WalkerTestCaseFixture *fixture,
       NULL,
       NULL);
 
-  rewind (fixture, walker, 8);
+  rewind_sync (fixture, walker, 8);
   get_events (fixture, walker, 0);
-  rewind (fixture, walker, 8);
+  rewind_sync (fixture, walker, 8);
   get_events (fixture, walker, 2);
-  rewind (fixture, walker, 8);
+  rewind_sync (fixture, walker, 8);
   test_get_events_text (fixture, walker, 8, 1263427261, "I'''");
-  rewind (fixture, walker, 3);
+  rewind_sync (fixture, walker, 3);
   test_get_events_text (fixture, walker, 5, 1263427261, "I'");
-  rewind (fixture, walker, 1);
+  rewind_sync (fixture, walker, 1);
   test_get_events_text (fixture, walker, 7, 1263427202, "11");
-  rewind (fixture, walker, 2);
+  rewind_sync (fixture, walker, 2);
   test_get_events_call (fixture, walker, 5, 1263404881, 1);
-  rewind (fixture, walker, 2);
+  rewind_sync (fixture, walker, 2);
   get_events (fixture, walker, 0);
   test_get_events_text (fixture, walker, 1, 1263404950, "9");
-  rewind (fixture, walker, 0);
+  rewind_sync (fixture, walker, 0);
   test_get_events_text (fixture, walker, 5, 1263254401, "5''");
-  rewind (fixture, walker, 1);
+  rewind_sync (fixture, walker, 1);
   test_get_events_text (fixture, walker, 8, 1263168065, "G'''");
-  rewind (fixture, walker, 7);
+  rewind_sync (fixture, walker, 7);
   test_get_events_text (fixture, walker, 7, 1263168065, "G'''");
   test_get_events_text (fixture, walker, 7, 1263168063, "E");
-  rewind (fixture, walker, 2);
+  rewind_sync (fixture, walker, 2);
   test_get_events_text (fixture, walker, 6, 1263168061, "C");
-  rewind (fixture, walker, 10);
-  rewind (fixture, walker, 0);
-  rewind (fixture, walker, 5);
+  rewind_sync (fixture, walker, 10);
+  rewind_sync (fixture, walker, 0);
+  rewind_sync (fixture, walker, 5);
   test_get_events_text (fixture, walker, 16, 1263168005, "4''");
-  rewind (fixture, walker, 3);
+  rewind_sync (fixture, walker, 3);
   test_get_events_text (fixture, walker, 6, 1263168004, "3");
-  rewind (fixture, walker, 1);
+  rewind_sync (fixture, walker, 1);
   test_get_events_text (fixture, walker, 6, 1263081661, "A");
 
   tpl_log_walker_get_events_async (walker, 2, get_events_cb, fixture);
@@ -397,24 +397,24 @@ test_rewind (WalkerTestCaseFixture *fixture,
       filter_events,
       NULL);
 
-  rewind (fixture, walker, 8);
+  rewind_sync (fixture, walker, 8);
   get_events (fixture, walker, 0);
-  rewind (fixture, walker, 8);
+  rewind_sync (fixture, walker, 8);
   get_events (fixture, walker, 2);
-  rewind (fixture, walker, 8);
+  rewind_sync (fixture, walker, 8);
   test_get_events_text (fixture, walker, 8, 1263427201, "10");
-  rewind (fixture, walker, 3);
+  rewind_sync (fixture, walker, 3);
   test_get_events_text (fixture, walker, 5, 1263254406, "8");
-  rewind (fixture, walker, 1);
+  rewind_sync (fixture, walker, 1);
   test_get_events_text (fixture, walker, 7, 1263168064, "F");
-  rewind (fixture, walker, 2);
+  rewind_sync (fixture, walker, 2);
   test_get_events_text (fixture, walker, 5, 1263168061, "C");
-  rewind (fixture, walker, 2);
+  rewind_sync (fixture, walker, 2);
   get_events (fixture, walker, 0);
   test_get_events_text (fixture, walker, 1, 1263168062, "D");
-  rewind (fixture, walker, 0);
+  rewind_sync (fixture, walker, 0);
   test_get_events_text (fixture, walker, 5, 1263168002, "1");
-  rewind (fixture, walker, 1);
+  rewind_sync (fixture, walker, 1);
   test_get_events_text (fixture, walker, 4, 1263081661, "A");
 
   tpl_log_walker_get_events_async (walker, 2, get_events_cb, fixture);
-- 
cgit v0.10.2

